### DISCUSSION TITLE
Is it correct that I need to provide `offline` scope event if access_type=offline is specified?

### DISCUSSION THREAD

**Discussion by saboteurkid** [ory/hydra#2575] [Q&A] [answered]
I'm doing PKCE with AppAuth [example](https://github.com/openid/AppAuth-JS.git).

Steps to reproduce:

1. Start Hydra quickstart
2. Create client with
  ```shell
      hydra clients create \
      --endpoint http://127.0.0.1:4445 \
      --id app123 \
      --secret 123456 \
      --grant-types authorization_code,refresh_token \
      --response-types code,id_token \
      --logo-uri https://i.picsum.photos/id/237/300/300.jpg?hmac=9iUR3VHqf0Y9abGyuPZTpEIxHJL0sSvyNtJtDIMSylM \
      --name MyApp  \
      --scope openid,offline \
      --callbacks http://127.0.0.1:5555/callback,http://127.0.0.1:5556/callback \
      --token-endpoint-auth-method none
  ```
3. Update config in AppAuth.
 - PORT = 5556
 - openIdConnectUrl = 'http://localhost:4444'
 - clientId = 'app123'
 - redirectUri = `http://127.0.0.1:${PORT}/callback`
4. Result: `Refresh Token is undefined`

Question: Do I need to ask user for `offline` permission in consent page? I assume with `access_type=offline` there is no need for that, I am wrong?

**Comment by aeneasr** [comment, marked as answer]
I don't believe that we support `access_type`. That is a Google specific parameter as far as I recall. Please use `scope=offline_access` to get a refresh token!
